docs/src/formulation-details.md

```diff
@@ -15,12 +15,12 @@ Applying the standard BIM voltage cross-product (sine and cosine) substitution t
 
 ## `SDPUBFForm`
 The BFM SDP relaxation as described in:
-- Gan, L., & Low, S. H. (2014). Convex relaxations and linear approximation for optimal power flow in multiphase radial networks. In PSSC (pp. 1–9). Wroclaw, Poland. https://doi.org/10.1109/PSCC.2014.7038399
+- Gan, L., & Low, S. H. (2014). Convex relaxations and linear approximation for optimal power flow in multiphase radial networks. In PSSC (pp. 1–9). Wroclaw, Poland. <https://doi.org/10.1109/PSCC.2014.7038399>
 Note that this formulation is complex-valued and additional steps are needed to implement this in JuMP.
```

Comment:
I think we can get a more maintainable solution to this by putting these links in the docstring when the type is defined, then autodoc will pick them up automatically.  If you want to call out specific doc strings items at this location, there is a way to do that as well.